IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -111,7 +111,10 @@ def main(args, im_name, run_script):
         )
 
         try:
-            subprocess.run(script, timeout=int(args.timeout_threshold))
+            subprocess.run(
+                script,
+                timeout=int(args.timeout_threshold) if args.timeout_threshold else None,
```

Comment:
Surely this if statement could be avoided by setting the argument type to int and default to None?

Reply:
if default is None, int(None) will run into TypeError

Reply:
you are correct. updated.